.trunk/trunk.yaml

```diff
@@ -27,10 +27,15 @@ lint:
     - git-diff-check
     - markdownlint@0.44.0
     - prettier@3.5.3
-    - tflint@0.56.0
+    - tflint@0.57.0
     - trivy@0.61.1
     - trufflehog@3.88.26
     - yamllint@1.37.0
+  definitions:
+    - name: tflint
+      environment:
+        - name: TFLINT_CONFIG_FILE
+          value: ${workspace}/.trunk/configs/.tflint.hcl
```

Comment:
Instead of doing this, do we just want to include it at the root of the project so that it can be picked up like other configs and isn't hidden in the trunk folder?

Reply:
I wondered about that. 

I looked at a other linter configs in the .trunk/configs folder, and thought bending the configurations to keep the convention of storing the configs in the same place would be valuable for downstream consumers.

![CleanShot 2025-05-23 at 09 02 34](https://github.com/user-attachments/assets/7492a7e8-99d2-4218-824a-d2738edea384)

Reply:
@Gowiem did you have an opinion on this approach? I am reviewing what else needs to be take care before merging this PR.

Reply:
Let's keep this for now and ship it, but two things for us in the future to follow up on: 

1. Let's make sure this gets picked up by the OSS sync tasks we have in taskit 
2. We have config spread around and our more important configs like `terraform-docs.yaml`, `aqua.yaml`, `.editorconfig` and similar are top level. I like that personally because in the future we could rip out trunk and not changes to these configs would need to be made.